packages/core/components/AnnotationFilterForm/SearchBoxForm/index.tsx

```diff
@@ -1,32 +1,73 @@
+import { ChoiceGroup } from "@fluentui/react";
+import classNames from "classnames";
 import * as React from "react";
 
+import ListPicker from "../../ListPicker";
+import { ListItem } from "../../ListPicker/ListRow";
 import SearchBox from "../../SearchBox";
 import FileFilter from "../../../entity/FileFilter";
 
 import styles from "./SearchBoxForm.module.css";
 
 interface SearchBoxFormProps {
     className?: string;
+    items: ListItem[];
     title?: string;
+    onDeselect: (item: ListItem) => void;
+    onDeselectAll: () => void;
+    onSelect: (item: ListItem) => void;
+    onSelectAll: () => void;
     onSearch: (filterValue: string) => void;
-    onReset: () => void;
     fieldName: string;
     defaultValue: FileFilter | undefined;
 }
 
 /**
  * This component renders a simple form for searching on text values
+ * or selecting indiviudal items via the ListPicker
  */
 export default function SearchBoxForm(props: SearchBoxFormProps) {
+    const [isListPicking, setIsListPicking] = React.useState(false);
+
     return (
-        <div className={props.className} data-is-focusable="true">
+        <div className={classNames(props.className, styles.container)}>
             <h3 className={styles.title}>{props.title}</h3>
-            <SearchBox
-                defaultValue={props.defaultValue}
-                onReset={props.onReset}
-                onSearch={props.onSearch}
-                placeholder={`Search by ${props.fieldName}`}
+            <ChoiceGroup
+                className={styles.choiceGroup}
+                label="Filter type"
+                defaultSelectedKey={isListPicking ? "list-picker" : "search-box"}
+                options={[{
+                    key: 'search-box',
+                    text: 'Search box',
+                },{
+                    key: 'list-picker',
+                    text: 'List picker',
+                    disabled: props.items.length === 0,
+                }]}
+                onChange={(_, selection) => {
+                    props.onDeselectAll();
+                    setIsListPicking(selection?.key === "list-picker");
```

Comment:
minor: This has small side effect that you can't edit the list. e.g., if you've selected via list picker and close the modal, and then re-open from edit, it opens back to search box, and then if you try to go back to list picker it clears